spec/index.bs

```diff
@@ -2812,6 +2812,9 @@ dictionary GPUExternalTextureDescriptor : GPUObjectDescriptorBase {
 
             1. Let |source| be |descriptor|.{{GPUExternalTextureDescriptor/source}}.
 
+            1. [=Check the usability of the image argument=](|source|).
+                If this throws an exception or returns `bad`, then throw an {{InvalidStateException}} and stop.
```

Comment:
IDK how exceptions are supposed to work in spec text. It seems confusing to allow stack-unwinding behavior in spec text.

I copied this text from step 4 in createImageBitmap: https://html.spec.whatwg.org/multipage/imagebitmap-and-animations.html#:~:text=DOMException-,Check%20the%20usability%20of,Let,-imageBitmap

Reply:
In `createImageBitmap` we wrap this like this because this method returns a Promise, so we need to catch any error and pass it through the Promise instead.  
You may prefer to look at canvas [drawImage](https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-drawimage)'s steps instead:

```suggestion
            1. Let |usability| be the result of [=checking the usability of the image argument=].
            1. If |usability| is `bad`, then throw an {{InvalidStateException}} and stop..
```

*(the suggestion is probably wrong regarding the linking)*

Reply:
Ahh, thank you for the explanation and reference! I will update this.